src/main/java/svc/data/citations/datasources/transformers/MunicipalityIdTransformer.java

```diff
@@ -32,6 +32,24 @@
 			return null;
 		}
 	}
+	
+	public HashableEntity<Municipality> lookupMunicipalityIdFromCourtId(Long courtId){
```

Comment:
Why did you choose to put this here? It's not transforming anything. This seems more appropriate in a `MunicipalityManager` or something

Reply:
See above comment.  Similar reasoning.